src/language-js/parse/typescript.js

```diff
@@ -58,7 +58,7 @@ function getParseOptionsCombinations(text, options) {
   ];
 }
 
-function parse(text, options) {
+function parse(text, options = {}) {
```

Comment:
Changes in this file seems unnecessary, let's not do unrelated changes.

Reply:
They're not *strictly* necessary but only because the typechecker isn't actually checking that `parse` here conforms to the signature given in `Parser.parse` (I guess because `createParser` has no type annotations and so isn't being checked?). If this function *were* getting typechecked, it would fail without the change I make here, just like https://github.com/prettier/prettier/pull/15770#discussion_r1424531564 does. I think it's therefore better to make the change.